app/code/Magento/Sales/Model/Order/Pdf/Creditmemo.php

```diff
@@ -182,9 +186,6 @@ public function getPdf($creditmemos = [])
             $this->insertTotals($page, $creditmemo);
         }
         $this->_afterGetPdf();
-        if ($creditmemo->getStoreId()) {
```

Comment:
I guess this `if` statement should be moved under `foreach` since the locale is still being emulated there.
Otherwise, we never revert it.

Reply:
@dmytro-ch : I don’t think so. This function is really useless. You can see in the invoice pdf : app/code/Magento/Sales/Model/Order/Pdf/Invoice.php
No “if” block like creditmemo.

Reply:
Hi @dmytro-ch :

Thank you. I have changed the source code. Please check it again.